srcpkgs/singularity/INSTALL

```diff
@@ -0,0 +1,9 @@
+case "${ACTION}" in
+post)
+	if [ ! -d var/lib/singularity/mnt/session ]; then
+		mkdir -p var/lib/singularity/mnt/session
+	fi
+	chown root:root var/lib/singularity/mnt/session
```

Comment:
Why is this necessary? Root should be the owner by default.

Reply:
Yes, wanted to be safe. Removed it.